### ISSUE TITLE
improve reliability

### ISSUE THREAD

**Issue by barney-NG** [barney-NG/piCAMTracker#4]
Last weekend we had a long testing weekend.
Grey sky and enough distance (~15m) but we had 5% misses on flying out.
camera: V1 + fisheye
Raspi: Pi 3
I need some motion data files from situations where the camera is failing.
(And we need a button to start debugging because I don't setup the full wifi environment every time I am flying)

**Comment by barney-NG** [comment]
I found two issues:
1) Bug: A too big delta area reject stored the wrong geometry. (Sometimes just a few parts of an object are recognized as moving. In this cases the resulting rectangle is small or divided in a couple of sub- rectangles. If the rectangles cannot be joined there is a good chance to loose track.)
2) Enhancement: (Maybe still a bug) Removing intersections may result in intersected rectangles. (I simply added a second loop to remove this intersections)

If this improves the issue with straight fly bys has to be checked outside. Currently I see a good improvement in the debug sessions.
